Directory.Build.props

```diff
@@ -1,14 +1,40 @@
 <Project>
 
-  <PropertyGroup Label="Targets">
+	<!-- Versioning property for builds and packages -->
+	<PropertyGroup>
+		<VersionPrefix>1.0.40</VersionPrefix>
```

Comment:
It seems we will run the release pipeline every time `Directory.Build.Props` changes which might be more frequent. This could potentially be fixed as follows:
1. Create a `Version.props` file that only has the version-related property group
```
<Project>
  <PropertyGroup>
    <VersionPrefix>1.0.40</VersionPrefix>
  </PropertyGroup>
</Project>
```
3. Import this file in `Directory.Build.props` as something like:
    `<Import Project="Version.props" />`
4. Change the YML trigger to `Version.props`